changelog/fix_correction_add_a_missing_word_in.md

```diff
@@ -0,0 +1 @@
+* [#12440](https://github.com/rubocop/rubocop/pull/12440): Correction: Style/ArrayFirstLast documentation typo ([@ishigo974][])
```

Comment:
This change is documentation only. Can you remove this changelog entry?

Reply:
I wasn't sure if I had to do it or not, thanks for your quick check. I've removed it